test/integration/render/scenarios/viewport/viewportfeatures-viz-properties/scenario.js

```diff
@@ -0,0 +1,34 @@
+const map = new CartoMap({
+    container: 'map',
+    background: 'black'
+});
+
+const source = new carto.source.GeoJSON(sources['points']);
+const viz = new carto.Viz(`
+    color: red
+    @v_features: viewportFeatures($numeric, $cat)
+    @augmentedSize: 20
+`);
+const layer = new carto.Layer('layer', source, viz);
+
+layer.addTo(map);
+
+layer.on('loaded', async () => {
+    const feature = viz.variables.v_features.value[0];
+
+    // Using viz properties, viz variables (and also feature props)
+    const duration = feature.numeric;
+    const augmentedSize = feature.variables.augmentedSize.value;
+
+    await feature.color.blendTo('blue', duration);
+    await sleep(duration);
+
+    await feature.width.blendTo(augmentedSize, 10);
+    await sleep(10);
```

Comment:
What about using the `updated` event to decide when to mark it as loaded? We are already doing that for the e2e tests: https://github.com/CartoDB/carto-vl/blob/3eaf73bf8d930640a646a708fcc6d9c5ad0e76b6/test/acceptance/e2e/blendTo/scenario.js#L27